src/extension.ts

```diff
@@ -75,7 +75,8 @@ export async function activate(context: ExtensionContext) {
   });
 }
 
-export function deactivate() {
+export async function deactivate(): Promise<void> {
+  return new Promise((_resolve, _reject) => {});
```

Comment:
`Promise.resolve()`

Reply:
Ugh, so If I call `resolve()`, then this goes back to not reporting the shutdown. Looks like the shutdown only got reported because the promise would not get handled :\

I'm not sure why the shutdown isn't being reported. It happens the same in vscode-xml & vscode-quarkus.